.reuse/dep5

```diff
@@ -0,0 +1,30 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: tools-python
+Source: https://github.com/spdx/tools-python
+
+Files:
+  tests/*.json
+  tests/*.xml
+Copyright: 2023 spdx contributors
+License: Apache-2.0
+
+Files: assets/*
+Copyright: 2023 spdx contributors
+License: Apache-2.0
+
+Files: *.md
+Copyright: 2023 spdx contributors
+License: Apache-2.0
+
+Files:
+  tests/*.json
+  tests/*.xml
+Copyright: 2023 spdx contributors
+License: Apache-2.0
+
+Files:
```

Comment:
These files are generated/derived from spdx-3-model, which is covered by the Community Specification License.